skyportal/tests/frontend/test_gcnevents.py

```diff
@@ -319,7 +319,7 @@ def test_observationplan_request(driver, user, super_admin_token, public_group):
     driver.click_xpath(submit_button_xpath)
 
     # wait for the observation plan to complete
-    time.sleep(15)
+    time.sleep(30)
```

Comment:
Sleeps are a big no-no in the test suite!

Reply:
@stefanv what do you do with things you run in the IOLoops?

Reply:
@stefanv ah timeouts. one sec.